label_studio/tasks/models.py

```diff
@@ -273,7 +283,12 @@ def resolve_uri(self, task_data, project):
                 storage = self.storage or self._get_storage_by_url(task_data[field], storage_objects)
                 if storage:
                     try:
-                        resolved_uri = storage.resolve_uri(task_data[field])
+                        proxy_task = None
+                        if flag_set('fflag_fix_all_lsdv_4711_cors_errors_accessing_task_data_short',
+                                    self.project.organization.created_by):
```

Comment:
just write: 
```
flag_set('fflag_fix_all_lsdv_4711_cors_errors_accessing_task_data_short', user='auto'):
```

Reply:
Here I just left this for now since I don't want to change too many things, as this is already delaying deployment.